client-spark/common/src/main/java/org/apache/spark/shuffle/writer/BufferManagerOptions.java

```diff
@@ -64,9 +64,9 @@ public BufferManagerOptions(SparkConf sparkConf) {
             RssSparkConfig.RSS_WRITER_PRE_ALLOCATED_BUFFER_SIZE.defaultValue().get());
     requireMemoryInterval = sparkConf.get(RssSparkConfig.RSS_WRITER_REQUIRE_MEMORY_INTERVAL);
     requireMemoryRetryMax = sparkConf.get(RssSparkConfig.RSS_WRITER_REQUIRE_MEMORY_RETRY_MAX);
-    LOG.info(RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key() + "=" + bufferSize);
-    LOG.info(RssSparkConfig.RSS_WRITER_BUFFER_SPILL_SIZE.key() + "=" + bufferSpillThreshold);
-    LOG.info(
+    LOG.debug(RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key() + "=" + bufferSize);
```

Comment:
We should use `if (LOG.isDebugEnabled())` ahead of `LOG.debug`.

Reply:
Thank you for your reminder, I will send a followup pr later.

Reply:
> We should use `if (LOG.isDebugEnabled())` ahead of `LOG.debug`.

If don't have the time consuming var, the log.debug without `LOG.isDebugEnabled()` the is OK.

Reply:
The `LOG.isDebugEnabled())` is not needed if you would use string interpolation:
```suggestion
    LOG.debug("{}={}", RssSparkConfig.RSS_WRITER_BUFFER_SIZE.key(), bufferSize);
```

Reply:
To make it simpler, we can always use `LOG.isDebugEnabled())` to guard `LOG.DEBUG`.